presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -0,0 +1,214 @@
+    public void testScalarSubqueryWithGroupBy()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testCorrelatedJoin()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testCorrelatedNonAggregationScalarSubqueries()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testCorrelatedScalarSubqueries()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testCorrelatedScalarSubqueriesWithScalarAggregation()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testCorrelatedScalarSubqueriesWithScalarAggregationAndEqualityPredicatesInWhere()
+    {
+        // TODO https://github.com/prestosql/presto/pull/2843
+    }
+
+    @Override
+    public void testDelete()
+    {
+        // TODO https://github.com/prestosql/presto/pull/3181
+    }
+
+    @Override
+    public void testShowColumns()
+    {
+        MaterializedResult actual = computeActual("SHOW COLUMNS FROM orders");
+
+        MaterializedResult expectedParametrizedVarchar = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("orderkey", "bigint", "nullable, encoding=auto, compression=default", "")
+                .row("custkey", "bigint", "nullable, encoding=auto, compression=default", "")
+                .row("orderstatus", "varchar", "nullable, encoding=auto, compression=default", "")
+                .row("totalprice", "double", "nullable, encoding=auto, compression=default", "")
+                .row("orderdate", "varchar", "nullable, encoding=auto, compression=default", "")
+                .row("orderpriority", "varchar", "nullable, encoding=auto, compression=default", "")
+                .row("clerk", "varchar", "nullable, encoding=auto, compression=default", "")
+                .row("shippriority", "integer", "nullable, encoding=auto, compression=default", "")
+                .row("comment", "varchar", "nullable, encoding=auto, compression=default", "")
+                .build();
+
+        Assert.assertEquals(actual, expectedParametrizedVarchar);
+    }
+
+    @Override
+    public void testDescribeOutput()
+    {
+        // this connector uses a non-canonical type for varchar columns in tpch
+    }
+
+    @Override
+    public void testDescribeOutputNamedAndUnnamed()
+    {
+        // this connector uses a non-canonical type for varchar columns in tpch
+    }
+
+    @Override
+    public void testCommentTable()
+    {
+        // Kudu connector currently does not support comment on table
+        assertQueryFails("COMMENT ON TABLE orders IS 'hello'", "This connector does not support setting table comments");
+    }
+
+    @Override
+    public void testUnionWithAggregation()
+    {
+        // TODO https://github.com/prestosql/presto/pull/3174
```

Comment:
That's fishy. Column pruning or lack thereof should not break a connector.

Should this one ref  https://github.com/prestosql/presto/issues/3170 too?